prow/config.yaml

```diff
@@ -14957,6 +15059,40 @@ periodics:
         defaultMode: 256
         secretName: ssh-key-secret
 
+- cron: '1 5 * * 0' # Run at 05:01 on sunday
```

Comment:
It's sunday..  so 7.

Reply:
sunday is 0 in standard cron, 7 is actually invalid here - https://crontab.guru/#0_0_*_*_0

Reply:
In the link you gave, 7 seems to also work (however the description below says it's non-standard). So to confiurm, it's 0 in your implementation right?

Reply:
https://godoc.org/gopkg.in/robfig/cron.v2#hdr-CRON_Expression_Format
Sunday is 0.

Reply:
Cool.. thanks.

Reply:
7 doesn't pass presubmit: 
```
2017/11/20 19:05:27 End of range (7) above maximum (6): 7
Could not load prow configs: invalid cron string 1 15 * * 7 in periodic ci-kubernetes-e2e-gke-large-correctness: End of range (7) above maximum (6): 7
exit status 1
```

Reply:
We should probably link to these docs somewhere in our repo.